assets/prototype/application/utilities/money/test/formatAmount.test.ts

```diff
@@ -0,0 +1,32 @@
+import { formatAmount } from '../';
+
+describe('formatAmount', () => {
+	it('checks for formatted amount to be a string', () => {
+		const decimalPlaces = 2;
+
+		const format = formatAmount(decimalPlaces);
+		const result = format(5);
+
+		expect(typeof result).toBe('string');
```

Comment:
Should we add here:
`expect(result).toBe('5.00')` ?

Reply:
just read that assertion is `checks for formatted amount to be a string`;
ignore the previous comment